include/measurement_kit/ooni/error.hpp

```diff
@@ -18,6 +18,7 @@ MK_DEFINE_ERR(MK_ERR_OONI(5), InvalidMandatoryValueError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(6), MissingRequiredHostError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(7), MissingRequiredUrlError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(8), CannotOpenGeoIpCountryDatabase, "")
+MK_DEFINE_ERR(MK_ERR_OONI(8), CannotOpenGeoIpCityDatabase, "")
 MK_DEFINE_ERR(MK_ERR_OONI(8), CannotOpenGeoIpAsnDatabase, "")
```

Comment:
@AntonioLangiu beware that above here you're defining three different times the same error code number.